src/RichTextEditor/index.js

```diff
@@ -90,6 +99,7 @@ const RichTextEditor = (props) => {
 
   const handleCopyOrCut = useCallback((event, cut) => {
     event.preventDefault();
+    if (canCopy && !canCopy(editor)) return;
```

Comment:
If you want to avoid having to check for `canCopy` before calling it, you can pass a default prop for it that is a function that returns `true` like we do with `canBeFormatted` on line 189. Then if a function isn't passed, that default one will be called and will always return true (ie. it can be copied and we won't hit the return on line 102).

Reply:
So in the deconstruct of `props`?

```js
const RichTextEditor = (props) => {
  const {
    canCopy || () => true,
    canKeyDown,
    augmentEditor,
    isEditable,
    canBeFormatted
  } = props;
```

Reply:
No, in the `defaultProps` object at the end of the file where we have:
```
RichTextEditor.defaultProps = {
  isEditable: () => true,
  canBeFormatted: () => true,
};
```